gui/freehand.py

```diff
@@ -359,6 +363,8 @@ def motion_notify_cb(self, tdw, event, fakepressure=None):
         pressure = event.get_axis(Gdk.AxisUse.PRESSURE)
         xtilt = event.get_axis(Gdk.AxisUse.XTILT)
         ytilt = event.get_axis(Gdk.AxisUse.YTILT)
+        viewzoom = tdw.scale
+        viewrotation = tdw.rotation
```

Comment:
Re the angle correction stuff for tilt at line 431 below (sorry, github doesn't allow me to comment there)

If we are capturing view rotation now and passing it to the brush engine, shouldn't that block be moved to the brush engine's stroke_to? The goal here is that brush authors get an ascension relative to the reference angle on the canvas, not the viewport. That needs to still be the case, but it would be better to move it into the brush engine at the same time we change its interface, given that we're changing what xtilt and ytilt mean.

With that in mind, shouldn't viewrotation be flipped too, if the view is mirrored? Same reason as in the correction calculation below ☺